README.md

```diff
@@ -30,6 +31,9 @@
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
 
+## 🚀 Live Demo <a name="live-demo"></a>
+
+https://santitoot.github.io/desktop/
```

Comment:
- [OPTIONAL] Well done 👏🏻 , you did an awesome job by implementing the changes as per the previous code reviewer's request 👍🏻 but I would recommend you use the alternative text for your link. Please correct it by following the below suggestion.

```suggestion
[Live Project](https://santitoot.github.io/desktop/)
```